tests/jest/mask.spec.js

```diff
@@ -11,6 +11,8 @@ test('strip-down functionality', async () => {
     expect(stripDown('(999) 999-9999', '716 2256108')).toEqual('7162256108')
     expect(stripDown('(999) 999-9999', '(716) 2256108')).toEqual('7162256108')
     expect(stripDown('(999) 999-9999', '(716) 2-25--6108')).toEqual('7162256108')
+    expect(stripDown('+1 (999) 999-9999', '7162256108')).toEqual('7162256108')
```

Comment:
```suggestion
    expect(stripDown('+1 (999) 999-9999', '+1 (716) 225-6108')).toEqual('7162256108')
```
this would be a more accurate test to what you wanted to "fix". Which will likely fail.

Reply:
Just to clarify, which bug qre you trying to fix here? The example you posted (maska) behaves exactly as x-mask.
If you type 1, this generates `+1` and not `+1 (1`.

Reply:
@SimoTod - The bug is the application of autocomplete values

Reply:
Ok, that is the first time you mention it.
I think you need to describe the problem clearly otherwise when the maintainer looks at the issue, he will not understand what you are trying to solve.

Are we saying that the other plugins work differently depending on you entering the number vs copying vs using autoconplete? In that case, it looks like x-mask does a better job at staying consistent. It also sounds like it would be a lot of work to make it work the way you like.

If it's causing pains in your application, you can maybe disable the autocomplete?

Reply:
Hey @SimoTod - Your feedback is fair, apologies for the lack of clarity.

If I can get a working solution would you accept a PR? 

As @ekwoka mentioned, as tests are passed, we might want to align on the spec (preference for Jest) of what we expect the mask to do for some of these more complex cases?